codes/quantum/qubits/qec-metrology.yml

```diff
@@ -0,0 +1,35 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+code_id: qec_metrology
+physical: qubits
+logical: qubits
+
+name: 'Quantum Error-Correction for Quantum Metrology'
+introduced: '\cite{doi:10.1103/PRXQuantum.2.010343}'
+
+
+description: 'Quantum metrology is the study of precisely estimating parameters that characterize the
+Hamiltonian of a quantum system. 
+
+Classically, the precision of estimation improves with the number of trials of an experiment, n, as \( \propto 1/\sqrt{n}\), which is known as the Standard Quantum Limit (SQL). The more fundamental bound set by quantum mechanics on the estimation of a single parameter is called the Heisenberg Limit (HL), where the error scales as \( n^{−1}\). Using quantum-enhanced techniques, such as entanglement with the use of cat-states, it is theoretical possible to achieve the Heisenberg Limit of precision in the absence of noise. However, in the presence of noise, the advantage in precision due to entanglement is cancelled out. Hence quantum error-correction is proposed as a technique to suppress noise to achieve the Heisenberg limit of precision.
+
+Zhou et al. \cite{arXiv:1706.02445} showed that if the signal Hamiltonian obeys certain conditions dependent on the noise, then a quantum error-correcting code can be constructed which suppresses the noise and allows for Heisenberg limit precision in parameter estimation. The conditions are both necessary and sufficient, and can be summarized as follows: the signal Hamiltonian must be outside the span generated by the Lindblad operators \( L_j\), their adjoints \( L_j^{\dagger}\), inner products \( L_j^{\dagger}L_k\) and the Identity operator. This is referred to as the Hamiltonian-Not-In-Lindblad-Span condition (HNLS). The optimal code in this case can be found using a semi-definite program.
```

Comment:
I think there's an extension paper to Ham-not-in-Kraus-span, which is the more relevant one here since we deal with error channels. Please mention this.